plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss

```diff
@@ -10,16 +10,16 @@
 	.woocommerce-profiler-heading__title {
 		font-style: normal;
 		font-weight: 500;
-		font-size: 32px;
+		font-size: 40px;
 		line-height: 40px;
 		text-align: center;
-		color: #000;
+		color: $gray-900;
 		margin-bottom: 12px;
 		padding-top: 0;
 
 		@include breakpoint( '<782px' ) {
-			font-size: 28px;
-			line-height: 36px;
+			font-size: 28px !important;
+			line-height: 36px !important;
```

Comment:
It seems that we can remove font-size and line-height from .woocommerce-profiler-heading__title so we don't need to add `!important` here.

https://github.com/woocommerce/woocommerce/blob/41a7be8826de1d044e9151120c5a661b9307ceb0/plugins/woocommerce-admin/client/core-profiler/components/heading/style.scss#L53-L54

Reply:
Thanks! Updated in https://github.com/woocommerce/woocommerce/pull/38709/commits/e50a11a0d07e75c3f6c4b89ebf6e412f135bd8f2